index.js

```diff
@@ -1,12 +1,12 @@
-module.exports = function Obz(filter) {
+module.exports = function Obz (filter) {
   let value = null
   const listeners = []
   let oncers = []
 
-  function trigger(nextValue) {
+  function trigger (nextValue) {
     value = nextValue
     let length = listeners.length
-    for (let i = 0; i < length && value === nextValue; i++) {
+    for (let i = 0; i < length && value === nextValue; i++) { // eslint-disable-line
```

Comment:
can confirm that removing `value === nextValue` here breaks tests.
standard is correct that `value`/`nextValue` are not mutated in this loop, however value can be mutated by `set` while these loops are running, so yeah